dali/operators/image/resize/resize_attr.cc

```diff
@@ -22,73 +22,96 @@
+the operator keeps the aspect ratio of the original image.
 This option is equivalent to specifying the same size for all dimensions and ``mode="not_larger"``.
-)", 0.f, true)
-  .AddOptionalArg<vector<float>>("max_size", R"(Limit of the output size - when resizing using
-``resize_shorter``, "not_smaller" mode or otherwise leaving some extents unspecified, some images
-with high aspect ratios might produce extremely large outputs. This parameter puts a limit to how
-big the output can become. This value can be specified per-axis or uniformly for all axes.
+)code", 0.f, true)
+  .AddOptionalArg<vector<float>>("max_size", R"code(Limit of the output size.
+
+When the operator is configured to keep aspect ratio and only the smaller dimension is specified,
+the other(s) can grow very large. This can happen when using ``resize_shorter`` argument
+or "not_smaller" mode or when some extents are left unspecified.
+
+This parameter puts a limit to how big the output can become. This value can be specified per-axis
+or uniformly for all axes.
 
 .. note::
   When used with "not_smaller" mode or ``resize_shorter`` argument, ``max_size`` takes
   precedence and the aspect ratio is kept - for example, resizing with
   ``mode="not_smaller", size=800, max_size=1400`` an image of size 1200x600 would be resized to
   1400x700.
-)",
-  {}, false)
-  .AddOptionalArg("subpixel_scale", R"(If true, fractional sizes (either directly specified or
-calculated) will cause the input RoI to be adjusted to keep the scale factor. Otherwise,
-the scale factor will be adjusted so that the source image maps to the rounded output size.)",
-  true)
-  .AddOptionalArg<vector<float>>("roi_start", R"(Origin of the input region of interest (RoI);
-must be specified together with ``roi_end``. The coordinates follow the tensor shape order
-(same as ``size``). The coordinates can be either absolute (in pixels, the default) or
-relative (0..1), depending on the value of ``relative_roi`` argument. If a RoI origin is greater
-than RoI end in any dimension, the region is flipped in that dimension.)", nullptr, true)
-  .AddOptionalArg<vector<float>>("roi_end", R"(End of the input region of interest (RoI);
-must be specified together with ``roi_start``. The coordinates follow the tensor shape order
-(same as ``size``). The coordinates can be either absolute (in pixels, the default) or
-relative (0..1), depending on the value of ``relative_roi`` argument. If a RoI origin is greater
-than RoI end in any dimension, the region is flipped in that dimension.))", nullptr, true)
-  .AddOptionalArg("roi_relative", R"(If true, RoI coordinates are relative to the input size, with
-0 denoting top/left and 1 denoting bottom/right)", false);
+)code", {}, false)
+  .AddOptionalArg("subpixel_scale", R"code(If True, fractional sizes, directly specified or
+calculated, will cause the input RoI to be adjusted to keep the scale factor.
```

Comment:
```suggestion
calculated, will cause the input ROI to be adjusted to keep the scale factor.
```
Rheka insisted on that.

Reply:
Done